spinedb_api/alembic/versions/02581198a2d8_create_entity_class_display_order_tables.py

```diff
@@ -0,0 +1,58 @@
+"""create entity class display order tables
+
+Revision ID: 02581198a2d8
+Revises: ca7a13da8ff6
+Create Date: 2024-07-10 12:13:10.690801
+
+"""
+
+from alembic import op
+import sqlalchemy as sa
+from spinedb_api.helpers import DisplayStatus
+
+# revision identifiers, used by Alembic.
+revision = "02581198a2d8"
+down_revision = "c55527151b29"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.create_table(
+        "entity_class_display_mode",
```

Comment:
This table could have been called just 'display_mode' I believe? Any chance we could rename it now?

Reply:
In the drawio diagram the table is indeed called 'display_mode' so I supposed 'entity_class_dislpay_mode' was decided between @PiispaH and @jkiviluo. Maybe there are plans for other display modes in the future?

I don't think renaming should pose any issues if we decide to do so. The table/feature is not used anywhere, yet.

In any case, I don't have strong feelings about the name.